fetcher/tar.py

```diff
@@ -99,18 +101,26 @@ def unpack(self, downloadFirst=False):
         update = tarfile.open(name=self.workspacefile, mode='r:*')
```

Comment:
@petrosagg Definitely a very good point.
@floion No. Actually all we need is to replace workspacefile by IO stream.